### PR TITLE
Code review changes

### PR DESCRIPTION
**Pull request by feroxi-s**
Changed two variable names and one constructor

### Pull Request

---------------------------------------------

#### Comments

### CODE REVIEW

File: src/main/java/edu/uc/inventorymanager/dto/Item.java [hunk 1]
```diff
@@ -1,12 +1,15 @@
 package edu.uc.inventorymanager.dto;
 
 import lombok.Data;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
 
+@RequiredArgsConstructor
 public @Data class Item {
-    private int id;
+    @NonNull private int id;
```
Comment (feroxi-s) [inline src/main/java/edu/uc/inventorymanager/dto/Item.java:9]: @NonNull tells @RequiredArgsConstructor that this field is one of the required arguments

File: src/main/java/edu/uc/inventorymanager/dto/Item.java [hunk 2]
```diff
@@ -1,12 +1,15 @@
 package edu.uc.inventorymanager.dto;
 
 import lombok.Data;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
 
+@RequiredArgsConstructor
```
Comment (feroxi-s) [inline src/main/java/edu/uc/inventorymanager/dto/Item.java:7]: @RequiredArgsConstructor creates a constructor besides the default one with no parameters

File: src/main/java/edu/uc/inventorymanager/dto/Item.java [hunk 3]
```diff
@@ -1,12 +1,15 @@
...
 public @Data class Item {
-    private int id;
+    @NonNull private int id;
     private String name;
-    private String description;
+    @NonNull private String description;
     private String location;
-    private int assigned_to;
-    private int status_id;
+    private int assignedTo;
```
Comment (feroxi-s) [inline src/main/java/edu/uc/inventorymanager/dto/Item.java:13]: Renamed according to Java naming conventions

File: src/main/java/edu/uc/inventorymanager/service/ItemServiceStub.java [hunk 4]
```diff
@@ -19,9 +19,7 @@ public void save(Item item) {
 
     @Override
     public Item fetchItemById(int id) {
-        Item item = new Item();
-        item.setDescription("Test Description");
-        item.setId(id);
+        Item item = new Item(id, "Test Description");
```
Comment (feroxi-s) [inline src/main/java/edu/uc/inventorymanager/service/ItemServiceStub.java:38]: Creating a constructor that does everything in one line reduces technical debt and makes the code cleaner

File: src/main/java/edu/uc/inventorymanager/dto/Item.java [hunk 5]
```diff
@@ -1,12 +1,15 @@
...
+    private int statusId;
```
Comment (feroxi-s) [inline src/main/java/edu/uc/inventorymanager/dto/Item.java:14]: Renamed according to Java naming conventions

**Comment by feroxi-s** [comment]
I'm not sure why it's not building in CircleCI. It builds fine in my environment. Maybe a dependency issue

**Review by reediculous456 — APPROVED** [review]

File: src/main/java/edu/uc/inventorymanager/dto/Item.java (hunk 3 above)
Comment (discospiff) [reply src/main/java/edu/uc/inventorymanager/dto/Item.java:13]: Good.

File: src/main/java/edu/uc/inventorymanager/service/ItemServiceStub.java (hunk 4 above)
Comment (discospiff) [reply src/main/java/edu/uc/inventorymanager/service/ItemServiceStub.java:38]: Hmmmm.  I'm curious to see a source for this.  I see default constructors and setters, or via dependency injection, fairly frequently.

**Comment by discospiff** [comment]
It looks like this has already been merged.  So... works for me!

**Pull request merged**